src/styles/navigation.scss

```diff
@@ -0,0 +1,55 @@
+.nav {
+  display: flex;
+  justify-content: center;
+  margin: 0;
+
+  &__list {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__item {
+    display: flex;
+    justify-content: center;
+    height: 60px;
+
+    &:not(:last-child) {
+      margin-right: 20px;
+    }
+  }
+
+  &__link {
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $primary-color;
```

Comment:
Ensure that the variable `$primary-color` is defined in your variables file. If it's not defined, this will cause a compilation error.